### ISSUE TITLE
URL Rewrites without placeholders do not work 

### ISSUE THREAD

**Issue by domi2120** [joseluisq/static-web-server#243]
**Describe the bug**
I have a config containing rewrites without capture groups like this:
``` toml
  [advanced]

  [[advanced.rewrites]]
  source = "/content/images/login.jpg"
  destination = "/content/images/brands/demo/login.jpg"
```
These work fine in version 2.19, but no longer seem to work in 2.20.

**To Reproduce**
1. configure a rewrite with static paths like
	``` toml
      [advanced]

	  [[advanced.rewrites]]
	  source = "/content/images/login.jpg"
	  destination = "/content/images/brands/demo/login.jpg"
	```
2. call the path to be rewritten

**Expected behavior**
The path should be rewritten to the destination value.

**Environment and Specs**
 - static-web-server: v2.20.0
 - OS: Ubuntu 22.04.2 , kernel: 5.15.0-76-generic
 - Docker: containerd://1.6.21
 - Client: Chrome 114.0.5735.199

**Additional context**
I believe this may have been caused by https://github.com/static-web-server/static-web-server/pull/235/files#diff-eeb37db412eaa35c05c4a14c09cba43f9aae572c5da79fa31a2170d2665e307c. It looks to me like the new behaviour requires capture groups to exist within the source.

**Comment by joseluisq** [comment]
This should impact URL Redirects as well. However, it should be easy to be fixed.

**Comment by joseluisq** [comment]
It should be fixed now. A new patch release should come soon.
In the meantime, you can test this using the `devel` Docker images.

- `joseluisq/static-web-server:devel`
- `joseluisq/static-web-server:devel-alpine`
- `joseluisq/static-web-server:devel-debian`

Thanks for reporting.

**Comment by joseluisq** [comment]
Resolved on [v2.20.1](https://github.com/static-web-server/static-web-server/releases/tag/v2.20.1)
